### PR TITLE
Created Intake, Motor, and imported Classes

### PR DESCRIPTION
**Pull request by GalaArc**

### CODE REVIEW

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 1]
```diff
@@ -1,18 +1,71 @@
...
 public class Intake extends Subsystem {
     
     // Put methods for controlling this subsystem
     // here. Call these from Commands.
-
+	private SpeedController intakeMotor;
+	
+	
+	/**
+	 * @author GalacticArc
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: Please remove @author tags

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 2]
```diff
@@ -1,18 +1,71 @@
...
+		public double getMovement(){
+			return movement;
+		}
+	} 
+	/**
+	 * @param power
+	 * Power of the intake motor cannot exceed one
+	 * If it does, it sets it back to power
+	 */
+	public void setMovementLimit(double power) {
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: A function that does this already exists in MathUtil

File: src/org/usfirst/frc/team321/subsystems/Shooter.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ *Shooter class for the shooter of the robot
+ */
+public class Shooter extends Subsystem {
+    
+    // Put methods for controlling this subsystem
+    // here. Call these from Commands.
+	
+	private SpeedController roboShooter1, roboShooter2;
+
+
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Shooter.java]: You have too much whitespace everywhere.

File: src/org/usfirst/frc/team321/subsystems/Shooter.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	
+
+/**
+ * Retrieves the motors used for the shooters from the RobotMap
+ */
+	
+	public Shooter(){
+		
+		roboShooter1 = new Talon(RobotMap.LEFT_SHOOTER);
+		roboShooter2 = new Talon(RobotMap.RIGHT_SHOOTER);
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Shooter.java]: Unclear wording. There are two shooters on our robot?

File: src/org/usfirst/frc/team321/subsystems/Shooter.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+		
+	}
+	/**
+	 * @param power
+	 * Motor power cannot exceed 1, sets it less than one if it does
+	 */
+	public void shootingLimit(double power){
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Shooter.java]: Also redundant

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 6]
```diff
@@ -1,18 +1,71 @@
...
+	 *Intake into the robot, outtake into the robot, and stall
+	 *Checks if the absolute value of the power is above one or not
+	 *Motor power set to 0.8 as a precaution
+	 *Preset MIN and MAX values for motors for future code
+	 */
+	
+	public enum IntakeMovement {
+		
+		
+		INTAKE(0.8), OUTTAKE(-0.8), STALL(0.0);
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: why is this capped at +- 0.8?

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 7]
```diff
@@ -1,18 +1,71 @@
...
+		//MAX_INTAKE(1.0), MIN_OUTTAKE(0.0)
+		
+		private double movement;
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: This is super vague. What is moving? You should change this to a more specific word.

File: src/org/usfirst/frc/team321/subsystems/Shooter.java (hunk 4 above)
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Shooter.java]: Nitpick - "robo" is unnecessary in the variable name and makes it longer than necessary for no added clarity.

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 8]
```diff
@@ -1,18 +1,71 @@
...
+	
+	public Intake(){
+		
+		intakeMotor = new Talon(RobotMap.INTAKE_MOTOR);
+		
+	}
+	
+	/**
+	 * @author linw5
+	 *Created enumerations values for the intake
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: Don't use past tense

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 9]
```diff
@@ -1,18 +1,71 @@
...
 
+import org.usfirst.frc.team321.robot.RobotMap;
+
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.Talon;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
- *
+ *Intake class for the intake of the boulders
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: You should make this class javadoc more specific.

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 10]
```diff
@@ -1,18 +1,71 @@
...
+	 */
+	
+	public Intake(){
+		
+		intakeMotor = new Talon(RobotMap.INTAKE_MOTOR);
+		
+	}
+	
+	/**
+	 * @author linw5
```
Comment (HenryDangPRG) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: Also how is this author tag different from your previous author tag?

**Comment by GalaArc** [comment]
Idk it was automatically stamped into there

**Pull request merged**